source/ScreensaverHero.hx

```diff
@@ -14,42 +14,50 @@ class ScreensaverHero extends FlxSprite
 	var black:Float = 0;
 	var startingPos = new FlxPoint(0, 0);
 
+	var maxVel = 150;
+	var minVel = -150;
+
+	static var heroWidth = 20;
+	static var heroHeight = 20;
+
+	static var screenWidth(default, never):Int = 360;
```

Comment:
Note, the default screen width is 640px. The color-changing functionality is still good, but this doesn't quite match what it seems to be.

I recommend fixing it (or using FlxG.width instead) but not required to call this assignment done.